documentation/asciidoc/accessories/camera/libcamera_options_still.adoc

```diff
@@ -0,0 +1,128 @@
+----
+
+This puts `libcamera-still` into timelapse mode where it runs according to the timeout (`--timeout` or `-t`) that has been set, and for that period will capture repeated images at the interval specified here. (`libcamera-still` only.)
+
+Example: `libcamera-still -t 100000 -o test%d.jpg --timelapse 10000` captures an image every 10s for about 100s.
+
+----
+	--framestart			The starting value for the frame counter <number>
+----
+
+When writing counter values into the output file name, this specifies the starting value for the counter.
+
+Example: `libcamera-still -t 100000 -o test%d.jpg --timelapse 10000 --framestart 1` captures an image every 10s for about 100s, starting at 1 rather than 0. (`libcamera-still` only.)
+
+----
+	--datetime			Use date format for the output file names
+----
+
+Use the current date and time to construct the output file name, in the form MMDDhhmmss.jpg, where MM = 2-digit month number, DD = 2-digit day number, hh = 2-digit 24-hour hour number, mm = 2-digit minute number, ss = 2-digit second number. (`libcamera-still` only.)
+
+Example: `libcamera-still --datetime`
+
+----
+	--timestamp			Use system timestamps for the output file names
+----
+
+Uses the current system timestamp (the number of seconds since the start of 1970) as the the output file name. (`libcamera-still` only.)
+
+Example: `libcamera-still --timestamp`
+
+----
+	--restart			Set the JPEG restart interval <number>
+----
+
+Sets the JPEG restart interval to the given value. Default is zero.
+
+Example: `libcamera-still -o test.jpg --restart 20`
+
+----
+	--keypress,	-k		Capture image when Enter pressed
+----
+
+This switches `libcamera-still` into keypress mode. It will capture a still image either when the timeout expires or the Enter key is pressed in the terminal window. Typing `x` and Enter causes `libcamera-still` to quit without capturing.
+
+Example: `libcamera-still -t 0 -o test.jpg -k`
+
+----
+	--signal,	-s		Capture image when SIGUSR1 received
+----
+
+This switches `libcamera-still` into signal mode. It will capture a still image either when the timeout expires or a SIGUSR1 is received. SIGUSR2 will cause `libcamera-still` to quit without capturing.
+
+Example:
+
+`libcamera-still -t 0 -o test.jpg -s &`
+
+then
+
+`kill -SIGUSR1 $!`
+
+----
+	--thumb,			Set thumbnail parameters <w:h:q>
+----
+
+Sets the dimensions and quality parameter of the associated thumbnail image. The defaults are size 320x240 and quality 70.
+
+Example: `libcamera-still -o test.jpg --thumb 640:480:80`
+
+----
+	--encoding,	-e		Set the still image codec <string>
+----
+
+Select the still image encoding to be used. Valid encoders are:
+
+* `jpg` - JPEG (the default)
+* `png` - PNG format
+* `bmp` - BMP format
+* `rgb` - binary dump of uncompressed RGB pixels
+* `yuv420` - binary dump of uncompressed YUV420 pixels.
+
+Note that this option determines the encoding and that the extension of the output file name is ignored for this purpose. (`libcamera-still` only.)
```

Comment:
Does this affect the extension of the filenames used by the `--datetime` and `--timestamp` options?